geode-core/src/test/java/org/apache/geode/internal/net/BufferPoolTest.java

```diff
@@ -175,4 +181,39 @@ public void checkBufferSizeAfterAcquire() throws Exception {
     assertThat(newBuffer.limit()).isEqualTo(15000);
   }
 
+  @Test
```

Comment:
Please convert to JUnit 5 and use system properties annotations from Pioneer.
https://junit-pioneer.org/docs/system-properties/

Reply:
done